src/Tooltip/Tooltip.js

```diff
@@ -412,7 +412,13 @@ module.exports = kind(
 				anchorLeft = window.innerWidth - moonDefaultPadding - pBounds.left - pBounds.width / 2 >= b.width;
 			}
 
-			offset = this.floating ? (acRight + moonDefaultPadding) : paRightDiff;
+			if (floating) {
+				offset = acRight + moonDefaultPadding;
+			} else {
+				offset = paRightDiff;
+				// we need to account for activator border widths if we are not floating
+				acBorders = dom.calcBoxExtents(this.activator.node, 'border');
```

Comment:
I chose to use `this.activator.node` instead of `this.activator.hasNode()` to save a function call, as we will always have called `this.activator.getAbsoluteBounds()` already, which will have already cached the node reference. I don't feel super strongly about this and can probably be convinced to change this to `hasNode()` for future safety.